.github/workflows/ci.yml

```diff
@@ -88,20 +88,6 @@ jobs:
                 curl -L -O https://tiker.net/ci-support-v0
                 . ./ci-support-v0
 
-                #if [[ "$DOWNSTREAM_PROJECT" = "sumpy" ]] && [[ "$GITHUB_HEAD_REF" = "no-tree-in-wrangler" ]]; then
-                if [[ "$GITHUB_HEAD_REF" = "no-tree-in-wrangler" ]]; then
-                  with_echo git clone "https://github.com/inducer/$DOWNSTREAM_PROJECT.git" -b wrangler-refactor
-                else
-                  with_echo git clone "https://github.com/inducer/$DOWNSTREAM_PROJECT.git"
-                fi
-
-                cd "$DOWNSTREAM_PROJECT"
-                echo "*** $DOWNSTREAM_PROJECT version: $(git rev-parse --short HEAD)"
-                edit_requirements_txt_for_downstream_in_subdir
-                export CONDA_ENVIRONMENT=.test-conda-env-py3.yml
-                # Avoid slow or complicated tests in downstream projects
-                export PYTEST_ADDOPTS="-k 'not (slowtest or octave or mpi)'"
-                build_py_project_in_conda_env
-                test_py_project
-
+                export PYTEST_ADDOPTS="-k 'not slowtest'"
```

Comment:
This was copied from `meshmode` which doesn't have the `not slowtest` part. Was that on purpose?

Reply:
Don't recall that being on purpose. :shrug: Happy to wait for it to cause a problem.

Reply:
Fair enough! I was thinking of some of those heavy 3D tests in pytential, but it's probably not that bad.